src/main/java/org/broadinstitute/consent/http/service/UserFileService.java

```diff
@@ -0,0 +1,90 @@
+package org.broadinstitute.consent.http.service;
+
+import com.google.cloud.storage.BlobId;
+import org.broadinstitute.consent.http.cloudstore.GCSService;
+import org.broadinstitute.consent.http.db.UserFileDAO;
+import org.broadinstitute.consent.http.enumeration.UserFileCategory;
+import org.broadinstitute.consent.http.models.UserFile;
+
+import javax.ws.rs.NotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+public class UserFileService {
```

Comment:
these service classes are not yet used (except in tests) but I logically assumed these would be the minimum necessary functions for UserFile to be a useful abstraction.